ssh/tunnel.py

```diff
@@ -16,8 +16,6 @@
 
 from pkgpanda.util import write_string
 
-LOGGING_FORMAT = '[%(asctime)s|%(name)s|%(levelname)s]: %(message)s'
```

Comment:
Also, as to the point of callers losing their log format. The only callers of this code are test_*.py files in this folder, dcos-launch, and integration tests. In integration tests, logging format is handled by conftest.py, the test_*.py scripts have their own format explicitly set, and this PR does that for dcos-launch